.github/workflows/docker-image.yml

```diff
@@ -2,8 +2,6 @@ name: Docker Image CI
 
 on:
   workflow_dispatch:
-  push:
```

Comment:
You removed `push` and `PR`, but we need them.
Maybe it's better to create two separate workflows (.yml) files.

One like the existing for `push` and `pull_request` the other for `dispatch` and `schedule` with ARM builds.